js/graphql.md

```diff
@@ -0,0 +1,1346 @@
+---
+---
+# The GraphQL Transform
+
+The GraphQL Transform is a library that simplifies the process of developing, deploying, and maintaining GraphQL APIs. With it, you define your API using the GraphQL Schema Definition Language (SDL) and can then use this library where to transform it into a fully descriptive cloudformation template that implements the API's data model.
+
+The Transform can be run as an independent library, however it is integrated into the Amplify CLI via the `API` category for you to use immediately.
```

Comment:
"The Transform can be run as an independent library, and it is also integrated into the Amplify CLI via the `API` category for you to use immediately."

More opinions